amazon_price_per_item.user.js

```diff
@@ -58,13 +67,46 @@
       parsePrice: parsePrice,
     };
   }
+
   // If we're not returning exports, run the gm script
   var ID = "gm_amazon_price_per_item";
   var DEBUG = false;
   if (!DEBUG && window.location.search.indexOf("appi=1")) {
     DEBUG = true;
   }
   var STYLE_ESTIMATED = "color:blue;";
+  //var CURRENCY_SIGN = "$";
+  var CURRENCY_MODE='USD'
+
+  var LOCALE=document.documentElement.lang //"en-US"
+  let segments = LOCALE.split('-'), country=segments[0], region=segments[1];
+  //var DECIMAL_SEPARATOR="."
+  var DECIMAL_SEPARATOR= new Intl.NumberFormat(LOCALE).format(1.1).replaceAll('1','')
+  //var THOUSANDS_SEPARATOR=",";
+  var THOUSANDS_SEPARATOR=new Intl.NumberFormat(LOCALE).format(1111).replaceAll('1','')
+  let currencies = {
+    fr: 'EUR',
+    en: 'USD'
+  }
+  CURRENCY_MODE=currencies[country]||'USD'
+
+  let i18n_strings = {
+    fr: {
+        item: 'unité',
+        estimated: function(perItem){ return `Environ ${perItem} par unité`}
+    },
+    en: {
+        item: 'item',
+        estimated: function(perItem){ return `Estimated ${perItem} per item`}
+    }
+  };
+  let i18n=i18n_strings[country]
```

Comment:
Should probably fall back if the country doesn't exist (like you did for `CURRENCY_MODE`)?

I have been writing this in ES5 because I'm stuck in my old ways, so if you're going to introduce `let` from ES6+, you might as well use `const` too, right?

```suggestion
  const i18n = i18n_strings[country] || i18n_strings.en;
```